src/BuildScriptGenerator/Python/PythonBashBuildSnippetProperties.cs

```diff
@@ -18,7 +18,8 @@ public class PythonBashBuildSnippetProperties
             bool enableCollectStatic,
             string compressVirtualEnvCommand,
             string compressedVirtualEnvFileName,
-            bool runPythonPackageCommand)
+            bool runPythonPackageCommand,
+            string pythonPackageWheelProperty = null)
```

Comment:
nit: is the `= null` part necessary?

Reply:
in that way we are saying it's an optional argument .. so if it doesn't get provided it will be passed as null .. so in most of the case it will be null ... because packaging will be used by terrapin mostly ..